### ISSUE TITLE
SuperTabBar - enable/disable tabs

### ISSUE THREAD

**Issue by nicoulaj** [flex-users/flexlib#62]
Originally filed by phahm...@gmail.com on 2007-11-26T18:42:44

Hi, i'm using the SuperTabBar but i want to enable and disable the tabs
from within the tabs. For example:

I have 4 tabs (A,B,C and D). By default tabs B,C,D are disabled and when i
click a button that is in tab A, tab B gets enabled and so on. Remembering
that the SuperTabBar is within a component and the content of each tab is a
diferent flex component.

Thanks,
Peter

**Comment by nicoulaj** [comment]
Updated by dmcc...@gmail.com on 2007-12-25T14:31:04

Removed label Type-Defect
Added label Type-Enhancement
Added label SuperTabBar

**Comment by nicoulaj** [comment]
Updated by dmcc...@gmail.com on 2007-12-25T22:55:41

tabs get their enabled state from the enabled state of the child in the TabNavigator
(this is the behavior of the TabNavigator, not the SuperTabNavigator). So to set a
tab to be enabled or disabled you toggle the enabled property of the child itself,
not the tab directly. 

Here's an example MXML file to show how this works:

I'm marking this as Invalid, since I don't think this is a bug at all.

Original ticket set owner to dmccune
Original ticket set status to Invalid (we converted to closed)
